administrator/components/com_media/resources/scripts/store/state.js

```diff
@@ -23,6 +23,6 @@ export default {
     selectedDirectory: options.providers[0].name + ':/',
     // The currently selected items
     selectedItems: [],
-    // The currently selected items
-    selectedItems: [],
+    // The state of create folder model
+    showCreateFolderModal: false
 }
```

Comment:
Every file should end with an empty line. As you modified it, I tough would be nice to fix that CS issue as well.